content/en/monitors/monitor_types/forecasts.md

```diff
@@ -15,91 +15,101 @@ further_reading:
+#### Advanced Options
 
-* You can change the forecasting algorithm to be used here. See the [Forecast algorithms](#forecast-algorithms) section for tips on how to choose the best algorithm for your use case. Each algorithm also has additional settings is described in the next section.
-* Datadog recommends using larger intervals between points to avoid having noise influence the forecast too much.
-* The number of deviations controls the width of the range of forecasted values. A value of 1 or 2 should be large enough to forecast most "normal" points accurately.
+Datadog automatically analyzes your chosen metric and sets several parameters for you. However, the options are available to edit under **Advanced Options (auto)**:
 
-Then, you can choose if the monitor requires a full window of data for evaluation. Setting it to "Requires" forces the monitor to ignore (show "No Data" as the monitor state) any series that doesn't have data going back to the start of the time range shown in the Evaluation Window graph.
+{{< img src="monitors/monitor_types/forecasts/advanced_options.png" alt="Advanced options" responsive="true" style="width:80%;">}}
 
-Complete all steps in the New Monitor form (**Say what's happening**, etc.) and click **Save** to create the Forecast monitor.
+| Option                     | Description                                                                                                             |
+|----------------------------|-------------------------------------------------------------------------------------------------------------------------|
+| [Algorithm](#algorithms)   | The forecast algorithm (`linear` or `seasonal`)                                                                         |
+| Model                      | The forecast model (`default`, `simple`, or `reactive`) for the linear algorithm                                        |
+| Seasonality                | The forecast seasonality (`hourly`, `daily`, `weekly`, or `monthly`) for the seasonal algorithm                         |
+| [Daylight&nbsp;savings][4] | Available for `seasonal` forecast monitors with `daily`, `weekly`, or `monthly` seasonality.                            |
+| [Rollup][5]                | The rollup interval&mdash;larger intervals between points avoid noise influence on the forecast.                        |
+| Deviations                 | The width of the range of forecasted values&mdash;a value of 1 or 2 is generally large enough for most "normal" points. |
 
-Both the Monitor Edit page and the Monitor Status pages provide "Historical Context" so that you can see how the metric behaved in the past. This should give some insight into what the forecast algorithm takes into account when predicting future values.
+##### Algorithms
 
-## Forecast algorithms
+The available forecast algorithms are `linear` and `seasonal`. 
```

Comment:
Should we use tabs here ? to make this page more compact?

Reply:
Tabs for `Linear` and `Seasonal`? Sure, I could try that.